tiny_dnn/core/kernels/deconv2d_op.h

```diff
@@ -0,0 +1,44 @@
+/*
+    Copyright (c) 2017, Taiga Nomi
```

Comment:
I think it is now _Copyright (c) 2013, Taiga Nomi and the respective contributors_